src/components/WalletDropdown/MiniPortfolio/Pools/index.tsx

```diff
@@ -99,43 +101,50 @@ function PositionListItem({ positionInfo }: { positionInfo: PositionInfo }) {
   }, [walletChainId, chainId, connector, toggleWalletDrawer, navigate, details.tokenId])
 
   return (
-    <PortfolioRow
-      onClick={onClick}
-      left={<PortfolioLogo chainId={chainId} currencies={[pool.token0, pool.token1]} />}
-      title={
-        <Row>
-          <ThemedText.SubHeader fontWeight={500}>
-            {pool.token0.symbol} / {pool.token1?.symbol}
-          </ThemedText.SubHeader>
-        </Row>
-      }
-      descriptor={<ThemedText.Caption>{`${pool.fee / 10000}%`}</ThemedText.Caption>}
-      right={
-        <>
-          <MouseoverTooltip
-            placement="left"
-            text={
-              <div style={{ padding: '4px 0px' }}>
-                <ThemedText.Caption>{`${formatNumber(
-                  liquidityValue,
-                  NumberType.PortfolioBalance
-                )} (liquidity) + ${formatNumber(feeValue, NumberType.PortfolioBalance)} (fees)`}</ThemedText.Caption>
-              </div>
-            }
-          >
+    <TraceEvent
+      events={[BrowserEvent.onClick]}
+      name={SharedEventName.ELEMENT_CLICKED}
+      element={InterfaceElementName.MINI_PORTFOLIO_POOLS_ROW}
+      properties={{ chain_id: chainId, pool_token_0: pool.token0, pool_token_1: pool.token1 }}
+    >
+      <PortfolioRow
+        onClick={onClick}
+        left={<PortfolioLogo chainId={chainId} currencies={[pool.token0, pool.token1]} />}
+        title={
+          <Row>
             <ThemedText.SubHeader fontWeight={500}>
-              {formatNumber((liquidityValue ?? 0) + (feeValue ?? 0), NumberType.PortfolioBalance)}
+              {pool.token0.symbol} / {pool.token1?.symbol}
```

Comment:
Is this change intended? From the PR description, looks like we're mostly concerned with the <TraceEvents /> component